build/lib/core/protein_reader.py

```diff
@@ -236,12 +232,12 @@ def __add_bond_to_openbabel(mol, atom_id_a, atom_id_b):
     b.SetBondOrder(1)
 
 
-def _init_openbabel_atom(table, mol, res, atom_sites, i):
+def _init_openbabel_atom(mol, res, atom_sites, i):
     """Initialize openbabel atom
 
     Args:
-        table (OBElementTable): Element table to translate element type
-            to element numbers.
+        # table (OBElementTable): Element table to translate element type
```

Comment:
Update the docstring whenever something has changed

Reply:
Done